tap_core/src/lib.rs

```diff
@@ -8,49 +8,16 @@
 
 use std::time::{SystemTime, UNIX_EPOCH};
 
-use ethereum_types::Address;
-use ethers::{signers::WalletError, types::SignatureError};
-use receipt_aggregate_voucher::ReceiptAggregateVoucher;
 use thiserror::Error;
 
 pub mod adapters;
 pub mod eip_712_signed_message;
+mod error;
```

Comment:
I think this needs to be made pub since the user of the lib would need to be able to handle the errors.

Reply:
it is re-exported from the root of the crate